public/app/features/correlations/Forms/ConfigureCorrelationSourceForm.tsx

```diff
@@ -37,23 +38,34 @@ export const ConfigureCorrelationSourceForm = () => {
   return (
     <>
       <FieldSet
-        label={`Configure the data source that will link to ${getDatasourceSrv().getInstanceSettings(
-          correlation?.targetUID
-        )?.name} (Step 3 of 3)`}
+        label={t(
+          'correlations.source-form.titel',
+          `Configure the data source that will link to ${getDatasourceSrv().getInstanceSettings(correlation?.targetUID)
+            ?.name} (Step 3 of 3)`
+        )}
```

Comment:
Check out the [i18n contributing docs](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#plain-js-usage): 

we should not use template strings/manual string concatenation to put variables into translatable phrases as that localises poorly into other languages (the template string also prevents the extraction script from working properly, which should produce warnings but it doesn't....)

Instead, this should be something like 

```ts
  const dataSourceName = getDatasourceSrv().getInstanceSettings(correlation?.targetUID)?.name;
  return (
    <>
      <FieldSet
        label={t(
          'correlations.source-form.title',
          `Configure the data source that will link to {{ dataSourceName }} (Step 3 of 3)`,
          { dataSourceName }
        )}
      >
```

(note the typo correction for the ID - titel -> title)